Dockerfile

```diff
@@ -1,29 +1,22 @@
-ARG VERSION=latest
-ARG OS=debian
-ARG CLI_NAME=atmos
-
-FROM cloudposse/geodesic:$VERSION-$OS
+# syntax=docker/dockerfile:1
+FROM alpine:3.19 as deps
 
 # Install terraform
-ARG TF_1_VERSION=1.2.3
-RUN apt-get update && \
-    apt-get install -y -u --allow-downgrades terraform-1="${TF_1_VERSION}-*"
-
-# Set Terraform 1.x as the default `terraform` in this container.
-# Other versions of TF can be installed + used via Spacelift before_init scripts.
-RUN update-alternatives --set terraform /usr/share/terraform/1/bin/terraform
+ARG TF_1_VERSION=1.5.7 # last release under Mozilla Public License
+ARG ALPINE_VERSION=1.52.0
+ARG TARGETARCH
 
-# Install Atmos
-RUN apt-get install -y --allow-downgrades atmos
+RUN apk add --no-cache wget unzip; \
+    wget -q "https://releases.hashicorp.com/terraform/${TF_1_VERSION}/terraform_${TF_1_VERSION}_linux_${TARGETARCH}.zip"; \
+    wget -q -O atmos "https://github.com/cloudposse/atmos/releases/download/v1.52.0/atmos_1.52.0_linux_${TARGETARCH}"; \
+    unzip "terraform_${TF_1_VERSION}_linux_${TARGETARCH}.zip"
 
-# Shell banner
-ENV BANNER="Atmos"
+FROM alpine:3.19 as runner
 
-# Set SHELL for `atmos terraform shell`
-ENV SHELL=/bin/bash
+RUN apk add --no-cache bash=5.2.21-r0 git=2.43.0-r0 openssh=9.6_p1-r0 jq=1.7.1-r0
```

Comment:
These are all the dependencies I could think we need to run terraform, but there may be more. We may wanna manually set this as the runner image for a few stacks to test. Probably `app`.

Reply:
Yeah, let's definitely test prior to rolling out to all stacks 👍